Projects/Server/Maps/MapLoader.cs

```diff
@@ -85,21 +88,28 @@ public static void Configure()
 
             stopwatch.Stop();
 
-            Utility.PushColor(failures.Count > 0 ? ConsoleColor.Yellow : ConsoleColor.Green);
-            Console.Write(failures.Count > 0 ? "done with failures" : "done");
-            Utility.PopColor();
-            Console.WriteLine(
-                " ({0} maps, {1} failures) ({2:F2} seconds)",
-                count,
-                failures.Count,
-                stopwatch.Elapsed.TotalSeconds
-            );
-
             if (failures.Count > 0)
             {
-                Utility.PushColor(ConsoleColor.Red);
-                Console.WriteLine(string.Join(Environment.NewLine, failures));
-                Utility.PopColor();
+                logger.Warning(
+                    "Map Definitions loaded with failures ({0} maps, {1} failures) ({2:F2} seconds)",
+                    count,
+                    failures.Count,
+                    stopwatch.Elapsed.TotalSeconds
+                );
+
+                if (failures.Count > 0)
```

Comment:
This if statement is not needed since it is inside of an if statement that already does that.

Reply:
True! changing it